regression-test/suites/tpch_sf1_p1/tpch_sf1/explain/test_q10.groovy

```diff
@@ -61,9 +61,9 @@ suite("test_explain_tpch_sf_1_q10") {
         check {
             explainStr ->
 		explainStr.contains("VTOP-N\n" + 
-				"  |  order by: <slot 24> <slot 23> sum(`l_extendedprice` * (1 - `l_discount`)) DESC") && 
+				"  |  order by: <slot 24> <slot 23> sum(<slot 53> * (1 - <slot 54>)) DESC") && 
```

Comment:
we have so many slot lost its SourceExpr, do we have a way to add them back to obtain a better explain string?